.changeset/early-countries-sleep.md

```diff
@@ -0,0 +1,7 @@
+---
+'@twilio-paste/clipboard-copy-library': major
+'@twilio-paste/core': minor
+'@twilio-paste/website': patch
```

Comment:
No need for a patch on website, we don't treat it as a public package.

As this acts as the changelog, if you could add just a little more detail to the text, maybe explaining what the library is designed to help with, that would be amazing

Reply:
ah gotcha, i think it auto-patched from the `yarn changeset` run, i can remove it

Reply:
didn't add too much tbh, i think the clipboard component will be more interesting than this one :-)

Reply:
No, that's a perfect amount of detail. Thanks for that